test/E2E/Browser.purs

```diff
@@ -0,0 +1,100 @@
+module Test.E2E.Browser
+  ( Mode(..)
+  , TestOptions(..)
+  , launchWithExtension
+  , parseOptions
+  ) where
+
+import Prelude
+
+import Control.Applicative (apply, (<$>))
+import Data.Foldable (fold)
+import Data.Maybe (Maybe(Just, Nothing), fromMaybe)
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Class (liftEffect)
+import Effect.Console (error)
+import Options.Applicative
+  ( Parser
+  , execParser
+  , help
+  , long
+  , metavar
+  , option
+  , showDefault
+  , str
+  , strOption
+  , switch
+  , value
+  , info
+  , fullDesc
+  )
+import Toppokki as Toki
+
+data TestOptions = TestOptions
+  { chromeExe :: Maybe String
+  , namiDir :: String
```

Comment:
As I also mentioned in a comment on `launchWithExtensions`, we should have a sum type representing the wallet choice

To avoid name clashes with our own `Wallet` type, you could have something like

```purescript
data WalletExt 
  = NamiExt Path
  | GeroExt Path
```

Where `Path` is some newtype around `String`s, representing the paths now held in `namiDir` and `geroDir`

This way both the extension to run and its associated directory can be held by `TestOptions` without any redundancies or non-exhaustiveness

Reply:
Also `TestOptions` in general needs more doc comments